pkg/controller/config/config_controller.go

```diff
@@ -313,18 +342,53 @@ func (r *ReconcileConfig) applyAddons(req reconcile.Request, cfg *op.Config) (re
 			Version: flag.TektonVersion})
 		return reconcile.Result{}, err
 	}
-	log.Info("successfully applied all resources")
+	log.Info("successfully applied all addon resources")
+
+	err := r.updateStatus(cfg, op.ConfigCondition{Code: op.AppliedAddons, Version: flag.TektonVersion})
+	return reconcile.Result{Requeue: true}, err
+}
+
+func (r *ReconcileConfig) applyNonRedHatResources(req reconcile.Request, cfg *op.Config) (reconcile.Result, error) {
```

Comment:
Applying the community tasks appears bit different than applying Red Hat `ClusterTasks`. In a sense RH Tasks are applied from directory and this one applies from URLs. Any particular reason to fragment the method of applying tasks in this way?

Reply:
this issue explains the difference https://issues.redhat.com/browse/SRVKP-548